README.md

```diff
@@ -84,6 +84,17 @@ Optional settings:
 
 Ariadne Codegen implements a plugin system that enables further customization and fine-tuning of generated Python code. It’s documentation is available separately in the [PLUGINS.md](https://github.com/mirumee/ariadne-codegen/blob/main/PLUGINS.md) file.
 
+### Standard plugins
+
+Ariadne Codegen ships with optional plugins importable from the
```

Comment:
Please leave text wrapping to user preference. We don't do hard linebreaks in  our markdown.

Reply:
Sorry, my bad. I use `markdownlint` and my editor does it for me, didn't think about the rest of the file. I changed this to be a single super long line now

Reply:
This line still has hard linebreak in it.

Reply:
🤦 My bad, sorry for being sloppy. Fixed now!